app/index.jsx

```diff
@@ -17,10 +32,11 @@ let App = React.createClass({
     this.setState({ selectedElementIndex: clickedLiIndex })
   },
   render: function () {
-    let featureTitle = constants.elements[this.state.selectedElementIndex]
+    const featureTitle = constants.elements[this.state.selectedElementIndex]
+    const Component = componentCode[this.state.selectedElementIndex]
```

Comment:
I think it's a good idea to change it, just to make it more obvious what it's supposed to be. It'll be more readable that way.

`Component` could mean anything, since react classes are also known as components.